### DISCUSSION TITLE
Tailwind Conflict with Ant.Design

### DISCUSSION THREAD

**Discussion by AariefMohideen** [tailwindlabs/tailwindcss#12976] [Help] [answered]
Is there any option to use the Utility Class alone instead of tailwind component.
Since we are facing issue with conflict while using Tailwind with Antd UI.

Example:
Attachment 1: ![image](https://github.com/tailwindlabs/tailwindcss/assets/24786425/80547f37-6581-4878-a737-bad751623052)
- In Attachment 1, the primary button component used from Ant.d, but the button background has been applied as transparent (this style applied from tailwind css) - Styled applied using element attribute.

Attachment 2: ![image](https://github.com/tailwindlabs/tailwindcss/assets/24786425/b0aaba9f-a4aa-4717-a6aa-8a2be6996521)
- In Attachment 2: the primary button component is displayed as expected when disable the style which applied from tailwind css element property.

Question: Do we have any way to use only tailwind utility class to avoid issue as mentioned above.

**Comment by wongjn** [comment, marked as answer]
You could consider [disabling preflight](https://tailwindcss.com/docs/preflight#disabling-preflight).

**Comment by AariefMohideen** [comment]
@wongjn Appreciate your time for this answer. Thanks you!
